src/photos.cs

```diff
@@ -1471,4 +1495,55 @@ interface PHLocalIdentifierMapping
 		[NullAllowed, Export ("error")]
 		NSError Error { get; }
 	}
+
+	[TV (16,0), Mac (13,0), iOS (16,0)]
+	[BaseType (typeof(NSObject))]
```

Comment:
```suggestion
	[BaseType (typeof (NSObject))]
```
nit: here and a few other places